apps/web/app/components/basic-page/basic-page-section-heading.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, Flex, styled } from 'leather-styles/jsx';
+import { LearnMoreLink } from '~/layouts/page/page';
+import { sanitizeContent } from '~/utils/sanitize-content';
+
+export const styleTokens = {
+  whiteSpace: {
+    preLine: 'pre-line',
+    nowrap: 'nowrap',
+    normal: 'normal',
+  },
+  textDecoration: {
+    underline: 'underline',
+    none: 'none',
+  },
+  borderStyle: {
+    solid: 'solid',
+    dashed: 'dashed',
+    dotted: 'dotted',
+    none: 'none',
+  },
+} as const;
```

Comment:
This isn't how to do styling in Panda. Panda will autocomplete and typecheck these values when added directly.